pkgs/development/python-modules/markdownsuperscript/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, markdown }:
+
+buildPythonPackage rec {
+  pname = "MarkdownSuperscript";
+  version = "2.0.0";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1dsx21h9hkx098d5azpw81dcz23rrgzrwlymwv7jri348q26p748";
+  };
+
+  propagatedBuildInputs = [ markdown ];
+
+  doCheck = false; # Tests depend on packages which are not available yet in nixpkgs
```

Comment:
@FRidh OK thanks. I suppose I should change the comment on the `doCheck = false;` then. What do you think of:

```
doCheck = false; # Did not manage to get the tests to work, see PR #26985 for details on the failed attempt.
```